go.mod

```diff
@@ -32,7 +32,7 @@ require (
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/prometheus/client_golang v1.14.0
 	github.com/spf13/afero v1.6.0
-	github.com/spf13/cobra v1.0.0
+	github.com/spf13/cobra v1.5.0
```

Comment:
It looks to me that this upgrade is not needed for this PR.

Also, there are new dependencies added that do not look relevant for the changes in the code.

Reply:
So I used `go get github.com/ethereum/go-ethereum@v1.13.2` and then `go mod tidy` to update the `go-ethereum` dependency. Everything else was updated automatically by the `go mod` system.

Reply:
Thanks.